fastapi-url-shortener/source_code_final/README.md

```diff
@@ -0,0 +1,67 @@
+# FastAPI URL Shortener
```

Comment:
Do we need those extra README files in the individual subfolders beside the one in the root folder?